sql/src/test/java/io/crate/integrationtests/CrossJoinIntegrationTest.java

```diff
@@ -35,10 +35,11 @@
 
 import static io.crate.testing.TestingHelpers.printRows;
 import static io.crate.testing.TestingHelpers.printedTable;
+import static org.hamcrest.Matchers.arrayContaining;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.core.Is.is;
 
-@ElasticsearchIntegrationTest.ClusterScope(minNumDataNodes = 2)
+@ElasticsearchIntegrationTest.ClusterScope(numDataNodes = 1, numClientNodes = 0) // was: minNumDataNodes = 2,
```

Comment:
there is a CrossJoinSingleNodeIntegrationTest which has those settings already.. please move the test and revert the change here